### ISSUE TITLE
Fan control seems to be taken over by bios control on v209

### ISSUE THREAD

**Issue by thefunkygibbon** [Rem0o/FanControl.Releases#2873]
**Describe the bug**
it seems that the fan control loses "control" of some of the fans on my system so i get high fan speeds and the noise which comes along with that for no good reason, with the little question mark next to some of the controls.  seemingly random.   have to reboot to get fan control to re-gain control.   nothing has changed in bios since it started happening.  and i've even tried to turn the fans smart stuff off in the bios to no avail. 
issue has only started happening in the last week or 2
I've got all fans set to "force control" and restarting FC doesn't help either. 

**Is there a log.txt file next to FanControl.exe with recent date entries?**
nothing of note in there

**Relevant hardware specs and setup**
msi tomahawk b450 max

**Comment by Rem0o** [comment]
FanControl doesn't have exclusive control or a lock of some sort on your fans.   Any other software or BIOS may override at any time.  You seem to be having such a case. There is nothing FanControl itself can do to prevent this.   Force-control will simply "override the override" when it happens, but it won't prevent it.

**Comment by thefunkygibbon** [comment]
yes i get that. but it only started happening on 209 and for example its sitting there saying "?" at 60% speed on a specific fan when it should be a lot less than that.  if i quit fan control and open openhardwaremonitor and can see in there that the speeds of the fan have changed and gone down to 30% (presumably what the bios is telling it to be at) and then change the speeds with openhardwaremonitor it tells me that the speeds of the fans are increasing to the 100% or 10% that i tell it to. 

so based on that, it feels that it is specifically something to do with FC.   granted it is maybe something to do with my specific motherboard or something. but

**Comment by Triksu** [comment]
I have a similar issue. Fan Control failed to initialize, retrying... Expected 26 sensors, found 6...

I think it started this week. It might have been a Windows update or it might be Logitech G Hub update, but I did no changed to my Fan Control settings or bios. I just noticed fans were a bit more noisy and opening fan control i got errors.

**Comment by thefunkygibbon** [comment]
I had that too, although not as extreme.  29 found out of 30 I think it was , not sure why it was ever that high though as I don't really think I have 30 points of temperatures/fan speeds which is what I assume they are meaning
